.gitignore

```diff
@@ -3,3 +3,6 @@ node_modules/
 _build/
 dev.db
 site_media/
+env
+mydatabase
+requests
```

Comment:
Although it usually doesn't cause any problems, source code files should always end with a newline: https://robots.thoughtbot.com/no-newline-at-end-of-file